src/XrdCl/XrdClFileStateHandler.hh

```diff
@@ -476,6 +476,25 @@ namespace XrdCl
                                  ResponseHandler                   *handler,
                                  time_t                             timeout = 0 );
 
+      //------------------------------------------------------------------------
```

Comment:
This should have simply changed the existing method and not introduced a whole new method. The actual change that was required was pretty trivial as opposed to adding a whole new definition.